### DISCUSSION TITLE
Hot question

### DISCUSSION THREAD

**Discussion by faizan2220** [drodil/backstage-plugin-qeta#60] [Q&A]
what is the logic behind hot question

**Comment by drodil** [comment]
Hey, thanks for the (hot 😁 ) question! It's basically questions ordered by trend score which is currently calculated here: https://github.com/drodil/backstage-plugin-qeta/blob/main/plugins/qeta-backend/src/database/DatabaseQetaStore.ts#L176

It may not be the most elegant solution but as with everything else, it can be improved over time. If you have ideas, feel free to share or even contribute to the logic behind hot questions :)
